app/views/calls/create.js.erb

```diff
@@ -1 +1,16 @@
-$(".new-call").modal('hide');
+
+
+// Fires on call complete POST
+
+// TODO: write if statement for url ending in /edit/
```

Comment:
Should we backlog this or are you expect to handle? If we're backlogging it, can you create an issue for it and remove the TODO?